static/nodejs_server/tiles/bar_tile.ts

```diff
@@ -118,8 +125,10 @@ export async function getBarTileResult(
     ) {
       legend = chartData.dataGroup[0].value.map((dp) => dp.label);
     }
+    const csvLabelHeader =
```

Comment:
why not using this in all modes?

Reply:
I worry that changing the header will affect bard because bard reads the csv and then generates text ..